packages/harmony/src/components/hover-card/HoverCardHeader.tsx

```diff
@@ -1,7 +1,4 @@
-import { IconButton } from '~harmony/components/button'
-import { Flex } from '~harmony/components/layout'
-import { Text } from '~harmony/components/text'
-import { IconArrowRight } from '~harmony/icons'
+import { Flex, IconButton, Text } from '@audius/harmony'
```

Comment:
in harmony dont reference it's index file, the ~ is okay, relative is best

Reply:
oh thought this was in web maybe because HoverCardBody is in web